samples/openapi3/client/petstore/go-experimental/go-petstore/api/openapi.yaml

```diff
@@ -1973,6 +1973,7 @@ components:
           type: string
     fruitReq:
       oneOf:
+      - type: "null"
```

Comment:
I've just added this "null" type here to have one scenario with the "null" type in OneOf. @spacether , are you ok with this?

Reply:
Yup, I am okay adding this too our sample spec